docs/privacy.md

```diff
@@ -0,0 +1,40 @@
+# Privacy policy
+
+## Data pipeline
+
+Clangd remote index server is building an index directly from public LLVM
+[source code](https://github.com/llvm/llvm-project/). The [indexer
+code](https://github.com/clangd/llvm-remote-index/blob/master/.github/workflows/index.yaml)
+is executed by GitHub Actions and can be monitored on [clangd/llvm-remote-index
+Actions](https://github.com/clangd/llvm-remote-index/actions) tab. 
+
+## User data
```

Comment:
it might be worth pointing users at https://github.com/clangd/llvm-remote-index/blob/master/deployment/entry_point.sh and DockerFile in this section, especially when talking about logging. to point out server works in a "redacted logging" mode, in which it only logs information you mentioned + error templates.